File: pbweb/pbweb-jar/src/main/java/it/csi/pbandi/pbweb/dto/rendicontazione/AllegatiQuietanzeDTO.java
/*******************************************************************************
* Copyright Regione Piemonte - 2024
* SPDX-License-Identifier: EUPL-1.2
******************************************************************************/

package it.csi.pbandi.pbweb.dto.rendicontazione;

import java.util.List;

public class AllegatiQuietanzeDTO extends QuietanzaDTO {

	static final long serialVersionUID = 1;

	private List<String> nomiAllegati;

	public List<String> getNomiAllegati() {
		return nomiAllegati;
	}

	public void setNomiAllegati(List<String> nomiAllegati) {
		this.nomiAllegati = nomiAllegati;
	}

}

File: pbwebwcl/src/app/gestione-integrazioni/commons/dto/allegati-quietanze-dto.ts
/*
 * Copyright Regione Piemonte - 2024 
 * SPDX-License-Identifier: EUPL-1.2 
*/

import { QuietanzaDTO } from "./quietanza-dto";

export interface AllegatiQuietanzeDTO extends QuietanzaDTO {
    nomiAllegati: Array<string>;
}